frontend/src/components/CarrousselNews/style.jsx

```diff
@@ -1,5 +1,20 @@
 import styled from "styled-components";
 
 export default styled.section`
-  height: 20vh;
+height: 60vh;
+background-image: url("https://www.themoviedb.org/t/p/w1920_and_h800_multi_faces/5P8SmMzSNYikXpxil6BYzJ16611.jpg");
+//cette image est l'image "cover de l'API a priori - elle a déjà été retaillée"
+//Dans cette 1è phase elle est "en dur" pour styler le layout.
+// dans App.jsx, cet élément est positionné en 1er pour que l'image parte d'en haut à gauche. 
+// Une div a été ajoutée pour enlgober le reste du contenu afin qu'il couvre cette image en background. 
```

Comment:
I don't like French comments, and I think your clients will ask you to write everything in English as well